Source/ios/Storyboard/UIResponder+TyphoonOutletTransfer.m

```diff
@@ -0,0 +1,126 @@
+////////////////////////////////////////////////////////////////////////////////
+//
+//  TYPHOON FRAMEWORK
+//  Copyright 2015, Typhoon Framework Contributors
+//  All Rights Reserved.
+//
+//  NOTICE: The authors permit you to use, modify, and distribute this file
+//  in accordance with the terms of the license agreement accompanying it.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#import "UIResponder+TyphoonOutletTransfer.h"
+#import "NSLayoutConstraint+TyphoonOutletTransfer.h"
+#import <objc/runtime.h>
+
+@implementation UIResponder (TyphoonOutletTransfer)
+
+- (void)transferFromView:(UIView *)view
+{
+    unsigned count;
+    objc_property_t *properties = class_copyPropertyList([self class], &count);
+    unsigned i;
+    for (i = 0; i < count; i++) {
+        objc_property_t property = properties[i];
+        const char *propName = property_getName(property);
```

Comment:
What is the difference between names `propName` and `propertyName`?  Maybe better `propertyName` and `propertyNameString`?